petrinet/scripts/julia/test.jl

```diff
@@ -0,0 +1,38 @@
+# Testing
+#########
+
+include("./ASKEMPetriNets.jl")
+using AlgebraicPetri
+using Catlab.CategoricalAlgebra, Catlab.Graphics
+using JSON
+
+# Load a model
+askemnet = ASKEMPetriNets.to_petri("../../examples/sir.json")
+
+askemnet.model |> to_graphviz
+
+# Do modifications to the model
+askemnet.model[1, :sname] = :M
+askemnet.model[1, :tname] = :infect
+
+# Update the properties and JSON with new modifications to structure
+ASKEMPetriNets.update!(askemnet)
+
+# Print new JSON
+JSON.print(askemnet, 2)
+
+typed_askemnet = ASKEMPetriNets.to_typed_petri("../../examples/sir_typed.json")
+
+typed_askemnet.model |> to_graphviz
+
+@assert is_natural(typed_askemnet.model)
```

Comment:
I get this error from Line 28. Not sure what this assertion is testing.

![Screenshot 2023-06-20 at 10 19 34](https://github.com/DARPA-ASKEM/Model-Representations/assets/1030720/7b9200d8-b34b-47cc-ba35-32b8f97d0aac)

Reply:
similarly probably coming from version mismatches